Documentation: note required trailing slash for REST-API URLs (#5863)
paperless-ngx/paperless-ngx: docs/api.md

- `custom_fields`: Array of custom fields & values, specified as
  `{ field: CUSTOM_FIELD_ID, value: VALUE }`

!!! note

>     Be aware that the URLs of all of these endpoints end with a `/`slash.

Reviewer: ```suggestion
    Note that all endpoint URLs must end with a `/`slash.
```
